src/CurrencyHelper.php

```diff
@@ -2,29 +2,39 @@
 
 namespace PodPoint\I18n;
 
+use Illuminate\Support\Arr;
 use NumberFormatter;
 
 class CurrencyHelper extends LocalizedHelper
 {
+    /*
+     * NumberFormatter will round up with 2 decimals only by default.
+     * Sometimes we can display up to 6 decimals of the monetary unit (ex: £0.106544) for energy prices.
+     */
+    const MAX_PRECISION = 6;
+
     /**
      * Return a value in the given currency formatted for the given locale.
      *
      * @param float|int $value
      * @param string $currencyCode
      * @param string $locale
      *
+     * @param int $precision
      * @return string
```

Comment:
we usually leave an empty line before @return

Reply:
my bad